packages/middleware-compression/src/resolveCompressionConfig.ts

```diff
@@ -1,19 +1,25 @@
+import { normalizeProvider } from "@smithy/util-middleware";
```

Comment:
The `@smithy/util-middleware` dependency needs to be added to `package.json`.

Reply:
Added in https://github.com/smithy-lang/smithy-typescript/commit/2a3d748440f4d586fa473bed3b7725bd004803bf